app/models/step_by_step_page.rb

```diff
@@ -131,6 +131,10 @@ def links_last_checked_date
     steps.map(&:links_last_checked_date).reject(&:blank?).max
   end
 
+  def should_show_required_prepublish_actions?
+    has_draft? && !scheduled_for_publishing? && !can_be_published?
```

Comment:
Is the latter part of this conditional (`&& !can_be_published?`) required?  `has_draft?` checks if the `draft_updated_at` is not equal to `published_at`, so I think this should be enough to determine whether the `draft_updated_at` date changed because it was published.

The `!scheduled_for_publishing?` is definitely required.

Reply:
I think this is still required because `can_be_published?` also checks for `steps_have_content?`. Something might have a draft but also have empty steps.

Reply:
Cool, that makes sense.  I didn't consider that scenario.